Xamarin/MvvmSeed/MvvmSeed/MvvmSeed.Android/Views/SplashView.cs

```diff
@@ -0,0 +1,19 @@
+using Android.App;
+using Android.Content.PM;
+using MvvmCross.Core.ViewModels;
+using MvvmCross.Droid.Views;
+using MvvmCross.Platform;
+
+namespace MvvmSeed.Android.Views
+{
+    [Activity(MainLauncher = true, NoHistory = true, ScreenOrientation = ScreenOrientation.Portrait, Theme = "@style/Theme.Splash")]
```

Comment:
LOL Actually I think Samsung already did that... @AlphaGit https://www.wired.com/2017/01/why-the-samsung-galaxy-note-7-kept-exploding/